repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/national-indicator-library.yaml

```diff
@@ -0,0 +1,260 @@
+---
+/content/documents/corporate-website/national-indicator-library:
+  /sample-indicator:
```

Comment:
It's up to you NIL guys but we have been putting each folder/doc in a separate yaml file, but you guys don't have to follow us on that if you don't want 🙂